### PR TITLE
WIP: ensime-runner

### PR DESCRIPTION
**Pull request by dimart**
up to now, the first part of #293 – split Ensime and Sbt so that we could scale them separately – is solved

### CODE REVIEW

File: balancer/src/main/resources/reference.conf [hunk 1]
```diff
@@ -4,8 +4,10 @@ com.olegych.scastie.balancer {
   old-snippets-dir = ./target/old-snippets/
 
   remote-hostname = "127.0.0.1"
-  remote-ports-start = 5150
-  remote-ports-size = 1
+  remote-sbt-ports-start = 5150
+  remote-sbt-ports-size = 1
+  remote-ensime-ports-start = 6150
```
Comment (MasseGuillaume) [inline balancer/src/main/resources/reference.conf]: make shure to set this too: https://github.com/scalacenter/scastie/blob/master/deployment/production.conf#L6

Comment (dimart) [inline balancer/src/main/resources/reference.conf]: good point

File: ensime-runner/src/main/scala/com/olegych/scastie/ensime/EnsimeActor.scala [hunk 2]
```diff
@@ -339,18 +328,18 @@ class EnsimeActor(system: ActorSystem,
   }
 
   override def receive: Receive = {
-    case EnsimeConfigResponse(sbtDir: Path) =>
+    case EnsimeConfigResponse(sbtDir: String) =>
```
Comment (MasseGuillaume) [inline ensime-runner/src/main/scala/com/olegych/scastie/ensime/EnsimeActor.scala]: This should be the file contents since it's a distributed process now.

**Comment by MasseGuillaume** [comment]
This is what I will do today:

* [ ] Rebase Webpack and Docker refactor
* [ ] Remove Sbt or Ensime specific code in LoadBalancer
* [ ] Add ensime-runner to Deployment.scala
* [ ] Undo ensime Snapshot (M4 is a stable release)

**Comment by MasseGuillaume** [comment]
Rebased and merged in https://github.com/scalacenter/scastie/pull/324/commits/5f5155dbd31b3d9b716e19618de01f00e604f6b2

Thanks!

Deploying Monday morning.

**Pull request closed without merging**